site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "What's new in Svelte: August 2022"
+description: "Changes to SvelteKit's `load` before 1.0, better Vite configurations, plus some new HTML and CSS properties!"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+There's lots to cover this month... with big changes are coming to SvelteKit's design before 1.0 can be completed. If you haven't already, check out Rich's Discussion, [Fixing `load`, and tightening up SvelteKit's design before 1.0 #5748](https://github.com/sveltejs/kit/discussions/5748).
+
+There's been lots of great discussion there over the last few days. If any of the comments resonate with you, feel free to drop an emoji on them 🤓
```

Comment:
can you drop an emoji since the discussion has been locked? I can, but am an admin on the repo. I'm not sure if others can

Reply:
No, users can't

Reply:
Thanks for the heads up! I wasn't aware :)